src/contrib/cluster/Akka.Cluster.Tools.Tests/Singleton/ClusterSingletonRestart2Spec.cs

```diff
@@ -31,7 +31,7 @@ public class ClusterSingletonRestart2Spec : AkkaSpec
                                                                                            akka.loglevel = INFO
                                                                                            akka.actor.provider = "cluster"
                                                                                            akka.cluster.roles = [singleton]
-                                                                                           #akka.cluster.auto-down-unreachable-after = 2s
+                                                                                           akka.cluster.split-brain-resolver.stable-after = 2s
```

Comment:
Bug fixes for this spec - after doing a thorough review of some the cases where we had a race condition, it all comes down to whether or not `_sys1` was able to fully get `MemberRemoved` or whether or not it became `Unreachable` first. Speeding up the downing provider's decision-making process both ensures that this member gets removed quickly AND ensures that the number of retries during the `HandOver` process get capped so the test can complete on-time.